.github/workflows/publish-snapshot.yml

```diff
@@ -41,3 +39,4 @@ jobs:
           ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
           ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.MAVEN_SIGNING_PRIVATE_KEY }}
           ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.MAVEN_SIGNING_PASSWORD }}
+        if: "${{ env.ORG_GRADLE_PROJECT_mavenCentralUsername != '' }}"
```

Comment:
shouldn't that one go up to line 12 right after `publish` (so basically replace `if: ${{ github.actor != 'dependabot[bot]' }}`

Reply:
That doesn't work because the environment variable doesn't exist there

Reply:
👍